packages/@aws-cdk/aws-appmesh/test/test.virtual-router.ts

```diff
@@ -230,6 +238,10 @@ export = {
           match: {
             prefixPath: '/path2',
           },
+          timeout: {
+            idle: cdk.Duration.seconds(10),
+            perRequest: cdk.Duration.seconds(10),
+          },
```

Comment:
No need for this (not used in an assertion).

Reply:
ack